crates/matrix-sdk-crypto/src/machine/tests/mod.rs

```diff
@@ -434,6 +414,32 @@ async fn test_room_key_sharing() {
     assert_eq!(room_key_updates[0].session_id, alice_session.session_id());
 }
 
+async fn send_room_key_to_device(
+    alice: &OlmMachine,
+    bob: &OlmMachine,
+    room_id: &RoomId,
+) -> OlmResult<(Vec<Raw<AnyToDeviceEvent>>, Vec<RoomKeyInfo>)> {
```

Comment:
might be worth a bit of documentation saying this sends *from* alice *to* bob. (Alternatively, I guess you could use better names for the users.)

Could you also document what it returns?

Reply:
Fixed in 5ba5ada0e8 (renamed params)